web/source/kmwuibutton.ts

```diff
@@ -581,13 +589,9 @@ if(!window['keyman']['ui']['name']) {
 
       "#KeymanWebControl #kmwico ul li#KMW_ButtonUI_KbdIcon {"+
         "border-bottom: solid 1px #ad4a28;"+
-      "} ";   
+      "} ";
 
-    // Initialize after KMW is fully initialized, if UI already loaded
-    keymanweb['addEventListener']('loaduserinterface',ui.Initialize);
```

Comment:
What's the reason for removing this?

Reply:
I'd tried a project-wide search for the event and turned up nothing.  Probably shouldn't have tried the search with quotes, though - the related events are defined at the bottom of `kmwuimanager.ts`.  Interestingly, it doesn't look like eliminating 'em caused any issues - and the functions that trigger them are even marked with an interesting tidbit:

https://github.com/keymanapp/keyman/blob/0a22acd0e47ec18f3b2b8f3e68c4df0c4a47e80d/web/source/kmwuimanager.ts#L44-L54

Note line 49 there.